src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -49,15 +44,43 @@ class HuiConditionalCard extends HTMLElement implements LovelaceCard {
         return hass.states[c.entity].state !== c.state_not;
       });
 
-    if (visible && this.card) {
-      this.card.hass = hass;
+    if (this._card) {
+      this._card.hass = this._hass;
     }
 
     this.style.setProperty("display", visible ? "" : "none");
   }
 
-  public getCardSize() {
-    return computeCardSize(this.card);
+  public getCardSize(): number {
+    return computeCardSize(this._card);
+  }
+
+  public setConfig(config: Config): void {
+    if (
+      !config.card ||
+      !config.conditions ||
+      !Array.isArray(config.conditions) ||
+      !config.conditions.every(
+        (c) =>
+          c.hasOwnProperty("entity") &&
+          (c.hasOwnProperty("state") || c.hasOwnProperty("state_not"))
+      )
+    ) {
+      throw new Error("Error in card configuration.");
+    }
+
+    this._card = createCardElement(config.card);
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._card) {
+      return html``;
+    }
+
+    return html`
```

Comment:
Should be able to just return the card

Reply:
```
return html`${this._card}`
```
I went ahead and tested that with an undefined and you are correct. Thanks.

Reply:
I was talking about `return this._card;`

Reply:
`this._card` is not a `TemplateResult`

Reply:
Just double checked and TS does not like that

Reply:
The `render` function needs to return `TemplateResult`. All other functions are placing things in a template and thus can be elements, strings etc. So we should adjust the method signature.